shared/desktop/yarn-helper/log-to-trace.tsx

```diff
@@ -154,20 +154,25 @@ const buildGood = (old, info) => {
   const s = buildEvent(old, 'B')
   const e = buildEvent(info, 'E')
   if (s.ts > e.ts) {
-    console.log('no time travelers allowed')
+    console.log('🛑 no time travelers allowed')
     return []
   }
   return [s, e]
 }
 
 const convertLine = isGUI ? convertGuiLine : convertCoreLine
-const lines = fs.readFileSync(logfile, 'utf8').split('\n')
+let lines = fs.readFileSync(logfile, 'utf8').split('\n')
+// to help debug a single line just override it here
+// lines = [
+// 'Line to debug',
+// ]
 let lastGuiLine = null
 const knownIDs = {}
 lines.forEach(line => {
   const info = convertLine(line)
   if (!info) return
 
+  // console.log(`DEBUG line type: '${info.type}' \n${line}\n${JSON.stringify(info, null, 2)}`)
```

Comment:
Should this be here?

Reply:
its just a useful debugging line. this isn't a prod script so /shrug